app/main.py

```diff
@@ -2,0 +1,13 @@
+def copy_file(command: str) -> None:
+    parts = command.split(" ")
+    if parts[0] == "cp" and len(parts) == 3:
+        source_file = parts[1]
```

Comment:
Consider using more descriptive variable names as per the checklist. For example, `source_file` and `destination_file` could be `source_filename` and `destination_filename` to clearly indicate they are filenames.